tests/aws/services/sqs/test_sqs.py

```diff
@@ -438,6 +438,17 @@ def test_send_message_batch_with_oversized_contents_with_updated_maximum_message
 
         snapshot.match("send_oversized_message_batch", response)
 
+    @markers.aws.validated
+    @markers.snapshot.skip_snapshot_verify(paths=["$..Error.Detail"])
```

Comment:
Based on the snapshot generated in `tests/aws/services/sqs/test_sqs.snapshot.json`, it doesn't appear inecessary to skip this verification as no `Error.Detail` field exists. 

If you're curious, there's a section in the contributing docs on [skipping verification of snapshot tests](https://github.com/localstack/localstack/blob/master/docs/testing/parity-testing/README.md#skipping-verification-of-snapshot-test) 💡

Reply:
@gregfurman Yes i ran the code against an sqs i created in my personal aws account and matched the response with the error message that i've updated in my code, but if necessary feel free to generate a snapshot and test on your end as well since it's my first contribution to the repo so still getting caught up with all the documentation :)

Reply:
@gregfurman Were you able to check the above? 😄

Reply:
Hey! Thanks for the follow-up. I just noticed there was no update to the `tests/aws/services/sqs/test_sqs.validation.json`. Did you notice this file update on your local working-tree when you generated your snapshot against AWS?

Otherwise, I tested your changes against AWS and all looks good 😄

Reply:
No i don't recall any changes happening to this file. Perhaps the way i generated the snapshot may be incorrect, I'm still catching up. From what i understand this file should automatically get an entry with the time on which i generated the snapshot? Let me know if you want me to manually push an entry into this file. Like you said, its working as expected against AWS 😄

Reply:
Ahh no worries. I'm happy to just push it from my side if you're happy with that?

Reply:
Yep works! 😄

Reply:
@gregfurman I totally forgot about this PR. Is it pending some changes/fixes before it can be merged?

Reply:
Ahh thanks for the ping. Should be good to merge 😄 Had to fix a failing test that were fixed after rebase.